service/evidence/evidence_store.go

```diff
@@ -40,28 +44,43 @@ func (s *Service) StoreEvidence(_ context.Context, req *evidence.StoreEvidenceRe
 
 	_, err = e.Validate()
 	if err != nil {
-		return nil, status.Errorf(codes.InvalidArgument, "invalid req: %v", err)
+		return resp, status.Errorf(codes.InvalidArgument, "invalid req: %v", err)
```

Comment:
I'm not sure but if we have an error (evidence is not accepted), we only should return the error (without `resp`), right? When I think about it, maybe we could have left out the `status` in the response message then.

Reply:
Yeah, I would return an error here and not have a `status` in the response, this is redundant otherwise. Keep the response empty for now.

Reply:
Ok, than I´ll leave it as it is.